### PR TITLE
FeelingLucky fix

### PR DESCRIPTION
**Pull request by brigand**
See #55

### CODE REVIEW

**Comment by brigand** [comment]
I tried running this on babelboot which is hosted in digitalocean and I got captcha'd. Not sure what to do about this.

File: package.json [hunk 1]
```diff
@@ -1,8 +1,9 @@
 {
-	"repository": "https://github.com/oftn/oftn-bot",
-	"dependencies": {
-		"bhttp": "^1.2.1",
-		"object.assign": "^4.0.3",
-		"twitter": "~0.1.19"
-	}
+  "repository": "https://github.com/oftn/oftn-bot",
```
Comment (ljharb) [inline package.json]: `s/  /\t/g` :-p

Comment (brigand) [inline package.json]: Tell that to `npm install --save`!

Will fix...

Comment (ljharb) [inline package.json]: I did. https://github.com/npm/npm/pull/4760

**Pull request closed without merging**